### ISSUE TITLE
Retrieve All Row version 2.0

### ISSUE THREAD

**Issue by aleresche** [paulomarquesc/AzureRmStorageTable#32]
Hello,

many thanks for developing this module. 

I am encountering an issue while using the Get-AzTableRow cmdlet.

documentation says : 
```
.EXAMPLE
# Getting all rows
Get-AzTableRow -Table $Table
```
I have an Azure storage table with over 70k entities. and if I use the following cmdlet and params 
```
Get-AzTableRow -Table $mytable
```
I only got 1k entities in return.

![testaztable](https://user-images.githubusercontent.com/20904217/55564215-b24fdd00-56f7-11e9-9180-9d2b923b582a.png)

is this working as intended ?  I would expect all entities to be return.

Many thanks for your help

Best Regards,

**Comment by paulomarquesc** [comment]
Hi @aleresche, currently there is a limitation from the platform perspective of how many rows you can retrieve in a single call an that's what you're getting into.

I'll investigate with the storage tables PMs to check if there is a workaround other then using filtering to bring the rows in batches.

**Comment by paulomarquesc** [comment]
Hi @aleresche, I just released version 2.0.2 that fixes this issue, it now supports retrieving more than 1000 rows.

**Comment by aleresche** [comment]
Hi @paulomarquesc, sorry for the late answer. Many many thanks for that fix :-) that's perfect
